spinnaker-monitoring-daemon/spinnaker-monitoring/__main__.py

```diff
@@ -112,10 +113,22 @@ def prepare_commands():
 
   return all_command_handlers, parser
 
+def disable_ssl_verification():
+  """"We must disable SSL verification when using self-signed certificates."""
+  try:
+    _create_unverified_https_context = ssl._create_unverified_context
+  except AttributeError:
+    # Legacy Python that doesn't verify HTTPS certificates by default
+    pass
+  else:
+    # Handle target environment that doesn't support HTTPS verification
+    ssl._create_default_https_context = _create_unverified_https_context
 
 def main():
   """The main program sets up the commands then delegates to one of them."""
 
+  disable_ssl_verification()
```

Comment:
Should this be controlled via --disable_ssl ?
If so add the option and for now call this something like prepare_ssl_verification(options) and check the option to disable. Or something else you think is better.